### ISSUE TITLE
Outdated instructions for Security groups creation

### ISSUE THREAD

**Issue by AvrumFeldman** [MicrosoftDocs/windowsserverdocs#5150]
On the current documentation the method to create the DHCP security groups is via `netsh`, while this is technically correct it is still outdated as there is already a PowerShell way to do it, via [Add-DhcpServerSecurityGroup](https://docs.microsoft.com/en-us/powershell/module/dhcpserver/add-dhcpserversecuritygroup).

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1a614930-18ea-fffb-62df-77ce38bf1393
* Version Independent ID: da1c1ecc-30f8-22f8-4ead-189574f823c7
* Content: [Deploy DHCP Using Windows PowerShell](https://docs.microsoft.com/en-us/windows-server/networking/technologies/dhcp/dhcp-deploy-wps)
* Content Source: [WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/networking/technologies/dhcp/dhcp-deploy-wps.md)
* Product: **windows-server**
* Technology: **networking**
* GitHub Login: @eross-msft
* Microsoft Alias: **lizross**

**Comment by illfated** [comment]
Fair point, that seems quite a bit backwards when the page title is "**Deploy DHCP Using Windows PowerShell**"...

BTW, do you have specific ideas on how much to replace in the existing page, like copy the example from the **Add-DhcpServerSecurityGroup** page and also add a link pointing to that page for details? Will it be required to delete the examples containing the `netsh` commands?

**Comment by AvrumFeldman** [comment]
@illfated I think your suggestion makes sense.

**Comment by illfated** [comment]
Fair enough. I just need to ponder a little bit how to phrase that change, as well as how much or how little to include in a PR. Feel free to pitch in with anything related that comes to your mind.

**Comment by IngridAtMicrosoft** [comment]
#assign:IngridAtMicrosoft 
gh issue comment 5132 -b #assign:IngridAtMicrosoft

**Comment by PRMerger10** [comment]
The following users don't have valid GitHub IDs, or are not collaborators on this repo: ingridatmicrosoft 
gh issue comment 5132 -b ingridatmicrosoft

**Comment by IngridAtMicrosoft** [comment]
#assign:IngridAtMicrosoft

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
